demisto_sdk/commands/secrets/secrets.py

```diff
@@ -72,8 +79,12 @@ def __init__(self, configuration=Configuration(), is_circle=False, ignore_entrop
     def get_secrets(self, branch_name, is_circle):
         secrets_found = {}
         # make sure not in middle of merge
-        if not run_command('git rev-parse -q --verify MERGE_HEAD'):
+        if self.input_path:
+            secrets_file_paths = self.input_path
+        else:
             secrets_file_paths = self.get_all_diff_text_files(branch_name, is_circle)
```

Comment:
if i understand correctly, this was inside the condition because we didnt want to get the diff if we're in the middle of the merge, now its not like that
why did you take it?